acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -12,6 +12,8 @@
 
   step 'add puppet user to switch' do
     switchs.each do |switch|
+      on(switch, "useradd -U puppet")
+      on(switch, "useradd -U missinguser")
```

Comment:
If we create the `missinguser` then we're subverting what the test is trying to do. I'm thinking it's better to just skip the test on arista.